I will copy tests from this:
https://github.com/nim-lang/Nim/pull/15342
https://github.com/nim-lang/Nim/issues/9754

Tests works as expected locally:
```
D:\QQPCmgr\Desktop\Nim>choosenim 1.2.6
   Switched to Nim 1.2.6

D:\QQPCmgr\Desktop\Nim>testament r stdlib/t9754.nim
FAIL: tests/stdlib/t9754.nim C
Test "tests\stdlib\t9754.nim" in category "stdlib"
Failure: reExitcodesDiffer
Expected:
exitcode: 0

Gotten:
exitcode: 1

Output:
{"age": 12, "bio": "Я Cletus", "blob": [65, 66, 67, 128], "name": "Cletus"}
true
true
alpha 100
omega 200
Traceback (most recent call last)
tparsesql.nim(107)       tparsesql
parsesql.nim(1502)       parseSQL
parsesql.nim(1494)       parseSQL
parsesql.nim(1169)       parse
parsesql.nim(1158)       parseStmt
parsesql.nim(1066)       parseSelect
parsesql.nim(944)        parseFromItem
parsesql.nim(1066)       parseSelect
parsesql.nim(944)        parseFromItem
parsesql.nim(1066)       parseSelect
parsesql.nim(580)        add
gc.nim(448)              newSeq
gc.nim(440)              newObj
alloc.nim(783)           rawAlloc
SIGSEGV: Illegal storage access. (Attempt to read from nil?)

FAILURE! total: 1 passed: 0 skipped: 0 failed: 1

D:\QQPCmgr\Desktop\Nim>testament --nim:bin/nim_temp r stdlib/t9754.nim
PASS: tests/stdlib/t9754.nim C                                     ( 7.26 sec)
Used bin/nim_temp.exe to run the tests. Use --nim to override.
```

--- a/tests/stdlib/t9754.nim
+++ b/tests/stdlib/t9754.nim
@@ -1,2 +1,6 @@
+discard """
+  joinable: false
+"""
+
 import tmarshal
 import tparsesql

--- a/tests/stdlib/t9754.nim
+++ b/tests/stdlib/t9754.nim
@@ -1,0 +1,2 @@
+import tmarshal
+import tparsesql

